src/components/EmojiPicker/EmojiPicker.vue

```diff
@@ -210,7 +208,7 @@ export default {
 			/**
 			 * Emits a object with more data about the picked emoji
 			 */
-			this.$emit('selectData', emojiObject)
+			this.$emit('select-data', emojiObject)
 
```

Comment:
Maybe a breaking change

Reply:
Yes, as it is it would be a breaking change. But I thought about reverting this one and telling eslint to ignore the line. I think it's not worth it having a breaking change and a new major for just two changed lines. We can easily include them the next time we really have to release a new major. What do you think?

Reply:
I made it non-breaking and disabled the rule on these two lines. I will create an issue, if we merge it like this, so we don't forget for the next major.

Reply:
Let's revert and add a comment so ESLint ignore it for now.
I created a PR so we do not forget it: https://github.com/nextcloud/nextcloud-vue/pull/2603.